### DISCUSSION TITLE
preact by default

### DISCUSSION THREAD

**Discussion by jose-donato** [vercel/next.js#17062] [Help]
I stumbled across preact (https://preactjs.com/) and saw the amazing benefits it can provide. I also saw that it can work with next in the [example](https://github.com/vercel/next.js/tree/canary/examples/using-preact).

What are the downsides? 

If there aren't any major downside has Next.JS consider to use it by default?

Greetings

**Comment by ljosberinn** [comment]
incompatibility with packages. e.g. emotion, see https://github.com/emotion-js/emotion/pull/896 for reference

**Reply by aulneau** [reply]
Emotion works fine with preact-compat. using both in many projects.

**Reply by ljosberinn** [reply]
do you happen to have a repo? investigated it in https://github.com/chakra-ui/chakra-ui/issues/2012 and `preact-render-to-string` was the culprit

**Reply by aulneau** [reply]
https://github.com/blockstack/docs is a next.js app with preact in prod and using emotion@11

**Reply by aulneau** [reply]
could be more an issue with chakra-ui vs preact/emotion

**Reply by ljosberinn** [reply]
well chakra is just @emotion/styled :/ theres nothing too special going on under the hood. thanks for the repo, I'll reinvestigate

**Reply by ljosberinn** [reply]
@aulneau thanks for the heads up - I tested quickly and it looks like some dep definitely changed - a barebones poc works! 🎉
